src/jit/compiler.cpp

```diff
@@ -49,6 +49,13 @@ AssemblyNamesList2* Compiler::s_pAltJitExcludeAssembliesList            = nullpt
 #endif // ALT_JIT
 
 /*****************************************************************************/
+#if defined(_TARGET_AMD64_) && defined(_MSC_VER)
+#include <intrin.h>
```

Comment:
Actually, this isn't target-specific, it's host-specific. So probably: (defined(_HOST_X86_) || defined(_HOST_AMD64_))

---
In reply to: [80571468](https://github.com/dotnet/coreclr/pull/7357#discussion_r80571468) [](ancestors = 80571468)